src/sdg/api/tests/test_producten.py

```diff
@@ -40,7 +42,6 @@ def setUp(self):
         self.referentie_organisatie = OverheidsorganisatieFactory.create(
             owms_identifier="https://www.referentie.com",
             owms_pref_label="referentie",
-            owms_end_date=None,
```

Comment:
By removing the explicit `owms_end_date`, the factory adds a future date, making the test pass. But, I think this is not correct. Please put back the explicit `None` here (and below) or remove it from the factory (as per lightning talk, we heard we should only add required fields)